app/derivative_services/av_derivative_service.rb

```diff
@@ -102,7 +102,7 @@ def cleanup_derivatives
 
   private
 
-    def audio_mime_types
+    def av_mime_types
```

Comment:
A third list of mime types? 😵‍💫 Is there anything we can do to align or combine these?